src/components/Counter.js

```diff
@@ -20,10 +20,10 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => {/* Fill me in */ }}>
+                <button onClick={() => {this.props.increment()}}>
```

Comment:
```suggestion
                <button onClick={this.props.increment}>
```
Because you're not passing any parameters, you don't need to actually make an arrow function here. You can just assign onClick to the function variable like so and it will work the same